benchmark/misc/startup-cli-version.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+const common = require('../common.js');
+const { spawnSync } = require('child_process');
+const path = require('path');
+const { Worker } = require('worker_threads');
+
+// This benchmarks the startup of various CLI tools that are already
+// checked into the source code. We use --version because the output
+// tends to be minimal and fewer operations are done to generate
+// these so that the startup cost is still dominated by a more
+// indispensible part of the CLI.
+const bench = common.createBenchmark(main, {
+  cli: [
+    'tools/node_modules/eslint/bin/eslint.js',
+    'deps/npm/bin/npm-cli.js',
+    'deps/corepack/dist/corepack.js',
+  ],
+  count: [30],
+});
+
+function spawnProcess(cli, bench, state) {
+  const cmd = process.execPath || process.argv[0];
+  while (state.finished < state.count) {
+    const child = spawnSync(cmd, [cli, '--version'], {
+      env: { npm_config_loglevel: 'silent', ...process.env },
+    });
+    if (child.status !== 0) {
+      console.log('---- STDOUT ----');
+      console.log(child.stdout.toString());
+      console.log('---- STDERR ----');
+      console.log(child.stderr.toString());
```

Comment:
are the console.logs here intentionally part of the benchmark? If so, a comment explaining the intent would be helpful so no one comes along later thinking they are extraneous.

Reply:
It's only here for debugging when there's an error (which normally shouldn't happen, and the benchmark would also error in that case). I've added a comment.